### ISSUE TITLE
Bulk User: Redo Do/Undo

### ISSUE THREAD

**Issue by nucleogenesis** [learningequality/kolibri#13609]
<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

❌ **This issue is not open for contribution. Visit <a href="https://learningequality.org/contributing-to-our-open-code-base/" target="_blank">Contributing guidelines</a>** to learn about the contributing process and how to find suitable issues.

<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

## Description

After some feedback on the workflow permitting the undoing of bulk user actions, the KModals will be removed and we will leverage the existing Snackbar pattern and componentry to provide the same capabilities.

<img width="720" height="288" alt="Image" src="https://github.com/user-attachments/assets/9c6db06b-8e28-46f5-bef4-3ac64b241f4d" />

## Acceptance criteria

This work is scoped to the [side panels in the users page](https://github.com/learningequality/kolibri/tree/develop/kolibri/plugins/facility/assets/src/views/users/sidePanels)

- [ ] All "Undo" KModals ([example](https://github.com/learningequality/kolibri/blob/3c56cfa48120da0f5ffc3fa30bf0d130c906aa4a/kolibri/plugins/facility/assets/src/views/users/sidePanels/EnrollLearnersSidePanel.vue#L4-L26)) should be removed.
- [ ] Introduce snackbar using the [`useSnackbar` composable](https://github.com/learningequality/kolibri/blob/develop/packages/kolibri/composables/useSnackbar.js) in the Users page
- [ ] When a user has succeeded in a bulk action, the user should have the associated `undo` logic executed by being passed as the `actionCallback` - the `actionText` should be the i18nized "Undo" message
- [ ] Update bulk user messages file - removing now-unused messages while adding the new ones per the Figma examples for each of the side panels

#### Accessibility requirements

- [ ] Ensure that the user's focus is put onto the "Undo" button whenever the Snackbar is presented with one.
- [ ] The user's focus should return to where it was before they initiated the bulk action workflow that resulted them being focused on the "Undo" button whenever they dismiss the Snackbar or navigate their focus away from it
- [ ] Snackbar should be dismissable w/ ESC
